sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -567,6 +567,19 @@ public void testBetweenCriteria() {
         assertThat(people).containsExactly(TEST_PERSON);
     }
 
+    @Test
```

Comment:
Can you please come up with some more test cases, so that we can avoid regression for this in future ?
For example, you can add few test cases for ReactiveCosmosTemplateIT in addition to CosmosTemplateIT. 

We generally prefer to add test cases for both sync and async APIs.

Reply:
@kushagraThapar sure, will do! 

Initially, I was a bit sceptical to add more because of the overhead incurred by integration tests. I was also considering adding unit tests but saw that there's a preference for integration tests throughout the project.

Reply:
@kushagraThapar added 808d3715bead16feaee9c502d670b67a92747b50